docs/src/pages/components/HorizontalGrid/WebHorizontalGrid.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Horizontal grid
+platform: web
+githubPath: bpk-component-grid
+sassdocId: grids
+---
+
+import BpkBlockquote from 'bpk-component-blockquote';
+import BpkBannerAlert, { ALERT_TYPES } from 'bpk-component-banner-alert';
+
+import Code from 'components/Code';
+import Readme from 'components/Readme';
+import PresentationBlock from 'components/PresentationBlock';
+import readme from 'bpk-component-grid/README.md';
+
+<BpkBannerAlert message={
```

Comment:
<img width="1046" alt="Screen Shot 2021-03-10 at 09 48 50" src="https://user-images.githubusercontent.com/73652/110610147-d50d8a80-8185-11eb-98a8-2914dd97a6c2.png">

It looks a bit tight in production, can we add a bit of margin to the top just to space it out a bit?

```suggestion
<BpkBannerAlert style={{marginTop: spacingSm }} message={
```

I think for a one off like this is it's fine to just inline the style instead of using a class, as we already know that if we do this more we'll make a proper deprecation warning component.

Reply:
Sure thing!